task-management/src/main/webapp/skin/css/tasks.less

```diff
@@ -2567,6 +2567,9 @@
           width: 100%;
         }
       }
+      .v-picker__body {
```

Comment:
I don't think we need to change on default Vuetify classes. could you check if the width property is useful in this case https://vuetifyjs.com/en/components/date-pickers/#width